packages/store/ts/config/v2/defaults.ts

```diff
@@ -5,8 +5,9 @@ export const CODEGEN_DEFAULTS = {
   userTypesFilename: "common.sol",
   outputDirectory: "codegen",
   // TODO: default to true if using top-level `namespaces` key (once its migrated to store)
-  namespaceDirectories: false,
-  indexFilename: "index.sol",
+  namespaceDirectories: false as boolean,
+  // @deprecated import table libraries from their individual files instead */
```

Comment:
this got added to defaults, not the input type

Reply:
Good spot, actually `indexFilename` isn't even on the input type